extensions/wikia/AdEngine/AdEngine2ExitstitialHooks.class.php

```diff
@@ -73,8 +73,7 @@ static private function handleExternalLink( $url, &$attribs ) {
 
 		if ( !$wgEnableOutboundScreenExt
 			|| $wgRTEParserEnabled    // skip logic when in FCK
-			|| empty( $wgTitle )        // setup functions can call MakeExternalLink before wgTitle is set RT#144229
-			|| $wgUser->isLoggedIn()  // logged in users have no exit stitial ads
+			|| empty( $wgTitle )      // setup functions can call MakeExternalLink before wgTitle is set
 			|| strpos( $url, 'http://' ) !== 0
```

Comment:
Wouldn't it be better to fully validate the URL? 
for example via using filter_var($url, FILTER_VALIDATE_URL)